src/Riok.Mapperly/Helpers/SymbolExtensions.cs

```diff
@@ -56,46 +57,25 @@ internal static bool TryGetEnumUnderlyingType(this ITypeSymbol t, [NotNullWhen(t
         return enumType != null;
     }
 
-    internal static IEnumerable<IMethodSymbol> GetAllMethods(this ITypeSymbol symbol) => symbol.GetAllMembers().OfType<IMethodSymbol>();
+    internal static IEnumerable<IMethodSymbol> GetAllMethods(this ITypeSymbol symbol, WellKnownTypes types) =>
+        types.GetAllMembers(symbol).OfType<IMethodSymbol>();
 
-    internal static IEnumerable<IMethodSymbol> GetAllMethods(this ITypeSymbol symbol, string name) =>
-        symbol.GetAllMembers(name).OfType<IMethodSymbol>();
+    internal static IEnumerable<IMethodSymbol> GetAllMethods(this ITypeSymbol symbol, string name, WellKnownTypes types) =>
+        symbol.GetAllMembers(name, types).OfType<IMethodSymbol>();
 
-    internal static IEnumerable<IPropertySymbol> GetAllProperties(this ITypeSymbol symbol, string name) =>
-        symbol.GetAllMembers(name).OfType<IPropertySymbol>();
+    internal static IEnumerable<IPropertySymbol> GetAllProperties(this ITypeSymbol symbol, string name, WellKnownTypes types) =>
+        symbol.GetAllMembers(name, types).OfType<IPropertySymbol>();
```

Comment:
I'd move all modified methods of this class and `GetFields` to the `WellKnownTypes` or a new class `SymbolAccessor` and use the symbol as parameter.